src/server/index.js

```diff
@@ -167,11 +167,11 @@ app.get('/login-callback',
       await User.save({
         auth0_id: req.user.id,
         // eslint-disable-next-line no-underscore-dangle
-        first_name: req.user._json.user_metadata.given_name,
+        first_name: req.user._json['https://spoke/user_metadata']['given_name'],
```

Comment:
will this break existing uses?  i.e. should we do something like:
 `const userMetadata = (req.user._json.user_metadata || req.user._json['https://spoke/user_metadata'] || {})`
... and then `userMetadata.given_name` ....